src/bgworker/mod.rs

```diff
@@ -117,6 +117,26 @@ fn session(worker: Arc<Worker>, mut handler: RpcHandler) -> Result<(), IpcError>
                 worker.call_destory(id);
                 handler = x.leave()?;
             }
+            RpcHandle::StatTuples { id, x } => {
```

Comment:
Can we merge `StatTuples`, `StatTuplesDone`, `StatSealed`, `StatGrowing` to one function? The info about write segment is useful too.

Reply:
done in new commit